conans/client/hook_manager.py

```diff
@@ -128,3 +130,38 @@ def _load_module_from_file(hook_path):
             sys.dont_write_bytecode = False
             sys.path.pop()
         return loaded
+
+    @staticmethod
+    def _import(filename):
```

Comment:
I still don't know why these changes are necessary. All of it seems a bit complex and probably a bit fragile. So far, those imports have been working fine. If there is an issue with the way the imports for hooks code are done, probably need to be treated as a separate bug report, and fixed in a separate PR, different to this issue. If it is a pre-condition for this feature, then we should review that first, of course.

Reply:
Sure!